src/utils.js

```diff
@@ -28,6 +28,10 @@ export function getDisplayName(WrappedComponent: string | Component): string {
 }
 
 export function validateSpacingProps(props: SpacingProps) {
+  if (isProd) {
```

Comment:
yeah this is removed out of `reflex.min.js` but not from `reflex.js` (since `reflex.js` is not uglified)